src/service/keys/keys.go

```diff
@@ -0,0 +1,147 @@
+package keys
+
+import (
+	"fmt"
+
+	"github.com/ConsenSys/orchestrate-hashicorp-vault-plugin/src/log"
+	"github.com/ConsenSys/orchestrate-hashicorp-vault-plugin/src/service/formatters"
+	usecases "github.com/ConsenSys/orchestrate-hashicorp-vault-plugin/src/vault/use-cases"
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+type controller struct {
+	useCases usecases.KeysUseCases
+	logger   log.Logger
+}
+
+func NewController(useCases usecases.KeysUseCases, logger log.Logger) *controller {
+	if logger == nil {
+		logger = log.Default()
+	}
+
+	return &controller{
+		useCases: useCases,
+		logger:   logger.Named("keys"),
+	}
+}
+
+// Paths returns the list of paths
+func (c *controller) Paths() []*framework.Path {
+	return framework.PathAppend(
+		[]*framework.Path{
+			c.pathKeys(),
+			c.pathImportKey(),
+			c.pathKey(),
+			c.pathSignPayload(),
+			c.pathNamespaces(),
+		},
+	)
+}
+
+func (c *controller) pathKeys() *framework.Path {
+	return &framework.Path{
+		Pattern:      "keys/?",
```

Comment:
The pattern corresponds to the value passed in the vault config?
like the following?
```
path "keys/*" {
  capabilities = ["read"]
}
```